### ISSUE TITLE
[MU4 Issue] Audio delay in note input mode when using MIDI keyboard 

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#14942]
**Describe the bug**
There is a perceptible delay (by my estimation, approx. half a second) between playing a note on my MIDI keyboard and hearing its playback when entering notes in note input mode.

**To Reproduce**
See demo video: 

https://user-images.githubusercontent.com/86290556/205617564-37aa582c-361b-4b59-b928-36bb698d593c.mp4

**Expected behavior**
Playback sound should occur together with the action of pressing a key on a MIDI keyboard. 

**Platform information**
 - macOS 12.6 (M1)
 - Roland F701
 - RC

**Additional context**
Seems to be the same issue as described in #14911, although that seemed to describe only "preview mode". I suspect, however, it is the same problem.
 
@Tantacrul FYI

**Comment by DmitryArefiev** [comment]
> Yeah, this is pretty serious. @DmitryArefiev - is this worse than before do you think?

@Tantacrul Yeah, I tried this on my Mac with RC build and USB MIDI Keyboard - the **latency got longer** comparing with older MU4 builds. And it become more longer comparing with MU3.

Now the latency level looks like this:
MU3->MU4->RC MU4
